Jenkinsfile_CNP

```diff
@@ -74,7 +74,15 @@ withPipeline("nodejs", product, component) {
             env.IDAM_OAUTH2_CLIENT_ID='sscs'
     }
 
-    onMaster {
+    //onAAT {
```

Comment:
Can we actually remove this? or if its needed add a comment to describe the circumstances?